lld/test/ELF/Inputs/dwarf5-debug-names-type-comdat.s

```diff
@@ -0,0 +1,26 @@
+	.section	.debug_info,"G",@progbits,1175092228111723119,comdat
```

Comment:
For newer tests we tend to move away from `Inputs/`. dwarf5-debug-names-type-comdat.s and the DWARF64 counterpart can be placed into `x86-64-dwarf5-64-debug-names-type-comdat.test`. Search for `.ifdef ERR` and `llvm-mc --defsym`. We can use some assembly preprocessing to deduplicate the input.